marlowe-playground-client/default.nix

```diff
@@ -31,7 +31,6 @@ let
     src = ./.;
     name = "marlowe-playground-client";
     psSrc = generated-purescript;
-    additionalPurescriptSources = [ "../web-common/**/*.purs" ];
```

Comment:
The reason this can be dropped is that all relevant sources are specified in `spago.dhall` and `${spagoPackages.buildSpagoStyle}` provides the right `purs` invocation including all of those paths.